docker-images/checksummer/Makefile

```diff
@@ -47,6 +47,6 @@ test:
 	--env AWS_BATCH_JOB_ID=1 \
 	--env CHECKSUM_ID=2 \
 	--env AWS_BATCH_JOB_ATTEMPT=1 \
-	$(IMAGE):$(VERSION) python /checksummer.py s3://org-humancellatlas-sam/even
+	$(IMAGE):$(VERSION) python /checksummer.py s3://org-humancellatlas-sam/even 342f31b73c44cbbd1dff188c4e881eed
```

Comment:
Could we add a comment here about what this id represents?

Reply:
will do